sql/core/src/test/java/test/org/apache/spark/sql/Java8DatasetAggregatorSuite.java

```diff
@@ -25,43 +25,50 @@
 
 import org.apache.spark.sql.Dataset;
 import org.apache.spark.sql.KeyValueGroupedDataset;
-import org.apache.spark.sql.expressions.javalang.typed;
```

Comment:
suggest to use rename a class on import, e.g. import org.apache.spark.sql.expressions.javalang.{typed => javaTyped}
instead of use full class name in rest of code

Reply:
It is Java not Scala. What do you propose doesn't work here.